src/NuGet.Clients/NuGet.VisualStudio.Implementation/Exceptions/AssetsFileMissingException.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NuGet.VisualStudio.Implementation.Exceptions
+{
+    internal class AssetsFileMissingException : InvalidOperationException
+    {
+        public AssetsFileMissingException() : base()
```

Comment:
Use new lines for these. See guideline 24 - https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md#c-coding-style